src/collections/news/2023-04-12-BBB-Intro.md

```diff
@@ -5,46 +5,15 @@ slug: bbb-intro
-the objectives of the project.
-
-[Blissymbolics](https://www.blissymbolics.org/), a universal symbol language originally developed
-by Charles K. Bliss for the purpose of international communication, has gained popularity as a
-method for assisting language development and communication skills to children with physical
-disabilities who are non-speaking. The origin of the use of Bliss as a bridge to communication and
-literacy can be traced back to the pioneer program initiated by Shirley McNaughton at the Ontario
-Crippled Children's Centre in 1971. Blissymbols offer a written semantic language without phonology
-that serves as a valuable alternative for nonvocal individuals, such as those with cerebral palsy,
-who face obstacles in acquiring phonological language skills.
-
-The rapid advancement of AI technology, particularly Large Language Models (LLMs), presents not only a
-tremendous opportunity to enhance societal engagement for nonvocal individuals but also comes with
-many risks for a group that is not recognized or included in the training data. The Baby Bliss Bot
-(BBB) project is a community-led investigation to explore the possibilities and challenges of
-leveraging AI to improve communication among Bliss users, while exploring alternative ways of training
-and tuning AI systems to serve individuals and groups who are outliers and marginalized minorities.
-
-The BBB project sets forth several key goals:
-
-* Investigating visual language systems and generative language models.
-* Exploring the process of co-learning with machine learning models.
-* Preserving the plasticity of formalized language or maintaining a living language while safeguarding
-its essence.
-
-The insights shared by AAC users underscore several crucial aspects:
-
-* Enabling Blissymbolics to serve as a semantic shorthand, expanding meaning effectively for unfamiliar
-listeners in a context-specific and adaptive manner.
-* Personalizing vocabulary, presentation, and input to meet the unique needs of each AAC user.
-* Assisting with the transition to literacy, particularly for second language learners.
-
-The Baby Bliss Bot project anticipates yielding valuable insights and specifications for the development
-of adaptive AI-based AAC systems. Alongside these findings, the project aims to create experimental
-prototypes, pushing the boundaries of communication accessibility for AAC users.
-
-By delving into the intersection of AI tools, language learning, and communication enhancement for AAC
-users, the Baby Bliss Bot project aspires to empower individuals utilizing Blissymbolics. Through
-collaborative exploration and innovation, the project strives to foster inclusivity and break barriers,
-ultimately improving the lives of nonvocal individuals within minority language systems.
+This Baby Bliss Bot project seeks to investigate the feasibility of utilizing AI tools to enhance communication
```

Comment:
I think it would be good to link to the project page here too. [Baby Bliss Bot Project]